### ISSUE TITLE
pretrain  performance 

### ISSUE THREAD

**Issue by Dbassqwer** [VirajBagal/MMBERT#4]
“For pretraining, we optimize the loss using the learning rate by a factor of 0.1 if the validation loss does Adam optimizer [14] with learning rate 2e − 5 and reduce not improve for 5 consecutive epochs”
----------------
Hi, I try to use the above parameters pretraining MLM on the ROCO dataset and finetune on VQA-RAD. The best model overall acc is only 66.9, if I don’t use pre-training, the overall acc has 66.7. 
How do you judge which pre-trained model we get is effective for downstream tasks? In addition, what is the probability of the token being masked you set?
Looking forward to your answer！

**Comment by zxd1874** [comment]
作者您好，我有一些关于这个项目的问题想请教您，可以要个您的联系方式嘛，这是我的QQ：616403663

**Comment by Control-xl** [comment]
> ## “For pretraining, we optimize the loss using the learning rate by a factor of 0.1 if the validation loss does Adam optimizer [14] with learning rate 2e − 5 and reduce not improve for 5 consecutive epochs”
> Hi, I try to use the above parameters pretraining MLM on the ROCO dataset and finetune on VQA-RAD. The best model overall acc is only 66.9, if I don’t use pre-training, the overall acc has 66.7. How do you judge which pre-trained model we get is effective for downstream tasks? In addition, what is the probability of the token being masked you set? Looking forward to your answer！

May I ask where the pretrained parameters come from? I fail to find it in this repo. Or do you just collect the pretraining data yourself to pretrain the model? I am also doubting about the high acc this model achieves.

**Comment by Dbassqwer** [comment]
> > ## “对于预训练，如果验证损失使用学习率 2e - 5 执行 Adam 优化器 [14] 并且在 5 个连续的 epoch 中减少没有改进，则我们使用学习率以 0.1 的因子优化损失”
> > 嗨，我尝试使用上述参数在 ROCO 数据集上预训练 MLM，并在 VQA-RAD 上进行微调。最好的模型整体acc只有66.9，如果我不使用预训练，整体acc只有66.7。你如何判断我们得到的哪个预训练模型对下游任务有效？另外，你设置的token被屏蔽的概率是多少？期待您的答复！
> 
> 请问预训练的参数是哪里来的？我没有在这个 repo 中找到它。或者你只是自己收集预训练数据来预训练模型？我也怀疑这个模型实现的高acc。

hyper-parameters

**Comment by Dbassqwer** [comment]
> > ## “For pretraining, we optimize the loss using the learning rate by a factor of 0.1 if the validation loss does Adam optimizer [14] with learning rate 2e − 5 and reduce not improve for 5 consecutive epochs”
> > Hi, I try to use the above parameters pretraining MLM on the ROCO dataset and finetune on VQA-RAD. The best model overall acc is only 66.9, if I don’t use pre-training, the overall acc has 66.7. How do you judge which pre-trained model we get is effective for downstream tasks? In addition, what is the probability of the token being masked you set? Looking forward to your answer！
> 
> May I ask where the pretrained parameters come from? I fail to find it in this repo. Or do you just collect the pretraining data yourself to pretrain the model? I am also doubting about the high acc this model achieves.

They don't provide pre-trained models. I use the hyper-parameters and ROCO dataset they describe in the paper for pre-training. But it doesn't seem to effective for downstream tasks.

**Comment by Dbassqwer** [comment]
> > ## “For pretraining, we optimize the loss using the learning rate by a factor of 0.1 if the validation loss does Adam optimizer [14] with learning rate 2e − 5 and reduce not improve for 5 consecutive epochs”
> > Hi, I try to use the above parameters pretraining MLM on the ROCO dataset and finetune on VQA-RAD. The best model overall acc is only 66.9, if I don’t use pre-training, the overall acc has 66.7. How do you judge which pre-trained model we get is effective for downstream tasks? In addition, what is the probability of the token being masked you set? Looking forward to your answer！
> 
> May I ask where the pretrained parameters come from? I fail to find it in this repo. Or do you just collect the pretraining data yourself to pretrain the model? I am also doubting about the high acc this model achieves.

They don't provide pre-trained models. I use the hyper-parameters and ROCO dataset they describe in the paper for pre-training. But it doesn't seem to effective for downstream tasks.

**Comment by Control-xl** [comment]
OK, thanks for your reply. I got it.
